gem2rpm/foreman_plugin.spec.erb

```diff
@@ -169,6 +169,7 @@ find %{buildroot}%{gem_instdir}/<%= spec.bindir %> -type f | xargs chmod a+x
 %{foreman_bundlerd_plugin}
 <% if has_assets -%>
 %{foreman_assets_plugin}
+%{foreman_assets_foreman}
```

Comment:
Won't this mess up cherry picks? Should we somehow require a certain foreman version or make the macro optional?

Reply:
The same applies to all "remove SCL code" changes we did recently, and we didn't bump stuff there?

Reply:
We could set that macro to nil in 3.4?
Non SCLed packages are not pickable further anyways

Reply:
I'd also be happy to pick the change back to 3.4. Then the risk is that the build fails with an unpackaged file, which is fine because CI shows it. Existing packages don't need a rebuild, but would benefit from it.

Reply:
That also works!

Reply:
Does this clear your concern about pickability?

Reply:
Yes it does. There's just a merge conflict now.

Reply:
How dare you!? ;-)
(rebased)